tables/utils.py

```diff
@@ -33,6 +33,12 @@
     '|': 'irrelevant',
 }
 
+np_byteorders = {
+    'big': '>',
+    'little': '<',
+    'irrelevant': '|'
```

Comment:
'irrelevant' is meant for ASCII strings and the like, where the order is, well, irrelevant.